docsource/releasenotes.rst

```diff
@@ -16,6 +16,7 @@ UPDATES:
   against Mongo 7.
 * Gradle has replaced Ant as the build tool. As a consequence, all the built artifacts are now
   located in the build directory, including the ``update_workspace_database_schema`` script.
+* A shadow jar is published on jitpack.io for supporting tests in other repos.
```

Comment:
Add a short explanation of what a shadow jar is so people can get the gist without having to google it.

Reply:
Added